iron/distribution.yaml

```
@@ -426,6 +426,21 @@ repositories:
       url: https://github.com/fictionlab/ros_aruco_opencv.git
       version: rolling
     status: maintained
+  aruco_ros:
+    doc:
+      type: git
+      url: https://github.com/pal-robotics/aruco_ros.git
+      version: humble-devel
+    release:
+      packages:
+      - aruco
+      - aruco_msgs
+      - aruco_ros
+      tags:
+        release: release/iron/{package}/{version}
+      url: https://github.com/pal-gbp/aruco_ros-release.git
+      version: 5.0.5-1
+    status: maintained
```

I'm going to suggest you add in a `source` entry here:

```suggestion
    source:
      type: git
      url: https://github.com/pal-robotics/aruco_ros.git
      version: humble-devel
    status: maintained
```

---

Done. Thank you for the suggestion!